util/config/gather-cray-prgenv-arguments

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+
+THIS_DIR=`dirname "$0"`
+CHPL_HOME="$THIS_DIR/../../"
+
+COMMANDS=`cc -craype-verbose 2>/dev/null`
+
+COMPILE=0
+LINK=0
+case $1 in
+  compile)
+    COMPILE=1
+    ;;
+  link)
+    LINK=1
+    ;;
+  *)
+    echo "Usage: script [compile|link]"
```

Comment:
Will you also add a brief description about what this script does to the usage or as a comment at the top of the file?